evals/runner.py

```
@@ -0,0 +1,307 @@
+    return CaseResult(
+        case["id"], case["category"], case["name"], True, f"status={r.status_code}",
+    )
+
+
+async def _run_ask(client: AgentClient, case: dict) -> CaseResult:
+    # Pre-seed state via direct DB writes (model never sees this).
+    if "setup" in case:
+        err = verify.apply_setup(case["setup"])
+        if err:
+            return CaseResult(
+                case["id"], case["category"], case["name"], False,
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)